### PR TITLE
Update wc_get_page_id compatability return type

### PR DESCRIPTION
**Pull request by johnolek**
#### What?

This updates the compatibility version of `wc_get_page_id` to always return an integer to match the behavior of WC core. The current behavior of returning a string could cause issues with themes/plugins that are expecting an integer value.

#### Tickets / Documentation

- [Current definition of wc_get_page_id](https://github.com/woocommerce/woocommerce/blob/9c83a07145ea5c451adc489f21eeaf895825b683/plugins/woocommerce/includes/wc-page-functions.php#L36-L57)

### CODE REVIEW

**Comment by karmatosed** [comment]
This looks like it doesn't have any conflicts with the current branch, but it would be great to work out how to move it on from sitting. Would it be able to get a review @chanceaclark?

**Review by chanceaclark — APPROVED** [review]
We need a changelog description for this change, otherwise looks good on my end.

**Comment by bc-erich** [comment]
@chanceaclark What is the merge process? Can we merge this or does the contributor merge this? We should document the merge process for future reference

**Comment by karmatosed** [comment]
@chanceaclark I am willing to take the merge on for this but wanted to know the steps so can also document them.

> We need a changelog description for this change, otherwise looks good on my end.

Would you like this added to this PR or a new PR? I am checking what your expectations are on the process.

**Comment by chanceaclark** [comment]
@bc-erich @karmatosed sorry to get back so late on this.

As soon as it's approved, any maintainer can go ahead and merge the PR unless there are any objections.

>Would you like this added to this PR or a new PR? I am checking what your expectations are on the process.

Good question, we really haven't had a process for this yet. For now, we can create the changelog when we need to cut a release, for example #461.

**Comment by karmatosed** [comment]
Thank you @chanceaclark based on your comments what I am going to do is the following:

1. Merge this.
2. Only create and add a line to changelog on release (which isn't something we are going go to do today).

**Pull request merged**
